contentctl/actions/detection_testing/infrastructures/DetectionTestingInfrastructure.py

```diff
@@ -1114,21 +1140,32 @@ def retry_search_until_timeout(
 
                     else:
                         empty_fields = empty_fields.union(current_empty_fields)
```

Comment:
I was keeping the `empty_fields` set around specifically because it was handy to check with the debugger while writing this & I was hesitant to remove what appeared to be load bearing. We can take it out if you want, but its a minimal amount of overhead computationally.